src/routes/blog/spyder-6-carlos/+page.md

```diff
@@ -0,0 +1,156 @@
+* [New improved comboboxes](https://github.com/spyder-ide/spyder/pull/21555)
+* [UI refinements to the Find pane](https://github.com/spyder-ide/spyder/pull/21622)
+* [Enhancements to hovers, hints & calltips](https://github.com/spyder-ide/spyder/pull/21710)
+* [Display & layout upgrades to Files/Project](https://github.com/spyder-ide/spyder/pull/21813)
+* [Remove redundant icons on OK, Cancel, etc](https://github.com/spyder-ide/spyder/pull/21945)
+* [Solutions to many other UI issues](https://github.com/spyder-ide/spyder/pull/21707)
+
+
+## Supporting the work of others
+
+![Spyder's revised About dialog, with tabs Overview, Community and Legal. Overview tab shows the IDE's name, version, website, major dependency versions and social media links, in an organized and comfortable layout](spyder6-about-dialog.png "Revised Spyder 6 About dialog, on which Carlos worked")
+
+Beyond my own projects, a large part of my role as a maintainer and leader is supporting the work of others.
+
+I've assisted [Juan-Sebsatian Bautista](https://github.com/jsbautista), one of our junior developers, in several significant enhancements to Spyder's UI.
+In particular, [we've made UI/UX improvements](https://github.com/spyder-ide/spyder/pull/21134) to layout and scaling of the empty pane display and added a clear message if the emptiness is due to the IPython Console.
+As part of that work, we also enhanced the About dialog layout as well.
+
+![Spyder Consoles with the New Console in Environment submenu displayed, showing new sections and user-added environments](console-envs-menu-improvements.png "UI improvements to the new console in environment menu")
+
+Additionally, I helped Juan-Sebsatian with [several significant improvements](https://github.com/spyder-ide/spyder/pull/22200) to the console environments menu and statusbar widget.
+These include showing user-added environments in the console envs menus for easy access, and breaking the consoles menu into sections by environment type for easier navigation.
+It also consolidates environment tracking functionality previously duplicated between the IPython Console and the Python Interpreter preference pane.
+
+![Spyder switcher dialog, with a text box and a list of matching files and other objects](file-switcher.png "Switcher in Spyder 6, with Carlos' UI improvements")
+
+I aided [Angela Remolina](https://github.com/AngelaRemolina)'s work on improving and adding real-time file search to the Switcher (`Ctrl-P`), making [a number of improvements](https://github.com/spyder-ide/spyder/pull/21275( to its performance and UI/UX.
+These changes make the switcher compute file results asynchronously in a separate process to avoid interface freezes and improve performance.
+Additionally, it highlights the matching characters in the results, filters the results to only include non-binary, text files that can reasonably be human-edited, clarifies the section layout and improves the tests.
```

Comment:
Oh, but you said under Description of Changes in the PR description

> * Highlight matched characters of Projects switcher results that match the searched text.

Also, "edited in the Editor" sounds rather repetitive, don't you think? 😉 

Without that change, it would be

```suggestion
Additionally, it highlights the matching characters in the results, filters the results to only include text files that can edited, clarifies the section layout and improves the tests.
```